### ISSUE TITLE
Make an LSTM meta-learning optimizer and train it using CIFAR10

### ISSUE THREAD

**Issue by mikolajsacha** [mikolajsacha/masters_thesis#2]

**Comment by mikolajsacha** [comment]
Use model from 'learning to learn by gradient descent by gradient descent' https://arxiv.org/abs/1606.04474
